src/averageAges.js

```diff
@@ -15,12 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const ableCentury = people.filter((person) => century
+    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
+    : person.sex === 'm');
```

Comment:
You are using the expression `person.sex === 'm'` twice, let's create a variable